### PR TITLE
Feature flask

### PR DESCRIPTION
**Pull request by patrickpierson**
adding some flask work for the "Next" section

### CODE REVIEW

File: flask.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+```bash
+export FLASK_APP=app.py
+flask run
+```
+
+Open your browser and go to http://localhost:5000
+
+You will see the following.
+
+```json
```
Comment (mblayman) [inline flask.md:87]: Nice use of json highlighting.

File: flask.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  return jsonify(weather_data)
+
+@app.route('/image')
+def image():
+  weather_data = weather()
+  image = icon(weather_data['weather'][0]['icon'])
+  return Flask.response_class(image, mimetype='image/png')
+```
+
+You will notice that the weather fuction looks familiar. We are adding a
```
Comment (mblayman) [inline flask.md]: fuction -> function

**Review by mblayman — APPROVED** [review]
Could you link to the /next/ page in some way? This is currently disconnected from the rest of the flow.

Aside from that and the typo, 👍

**Comment by patrickpierson** [comment]
@mblayman thoughts?

**Comment by mblayman** [comment]
LGTM

**Pull request merged**